marklogic-data-hub-central/ui/src/components/entities/merging/merging-step-detail/merging-step-detail.tsx

```diff
@@ -242,14 +244,14 @@ const MergingStepDetail: React.FC = () => {
         maxSources: i["maxSources"],
         default: i["default"] === true ? <FontAwesomeIcon className={styles.defaultIcon} icon={faCheck} data-testid={"default-" + i["strategyName"] + "-icon"} /> : null,
         priorityOrder: i.hasOwnProperty("priorityOrder") ? true : false,
-        delete: <Tooltip title={commonStrategyNames.indexOf(i["strategyName"]) !==-1 ? MergeStrategyTooltips.delete : ""}>
-          <FontAwesomeIcon
+        delete: <HCTooltip text={commonStrategyNames.indexOf(i["strategyName"]) !==-1 ? MergeStrategyTooltips.delete : ""} id="delete-strategy-tooltip" placement="top">
+          <i><FontAwesomeIcon
```

Comment:
@federicoRufrancosMarklogic  I see in different files that you  wrap `<FontAwesomeIcon />` within  `<i>`  without any `aria-label`.
Is it really necessary to have `<i/>`?

Reply:
@romanlescano  I need to add the empty <i>, otherwise the tooltip doesn't show up.
And the aria-label was needed on those cases the there was a running test which need to access the tooltip